### ISSUE TITLE
eol-last errors reported on the wrong line

### ISSUE THREAD

**Issue by Standard8** [BenoitZugmeyer/eslint-plugin-html#63]
Using this code:

```html
<html>
<head>
  <title>Test page</title>
</head>
<body>
  <script>document.write("Hello world!");</script>
</body>
</html>
```

I get the output:

```
test.html
  9:1  error  Newline required at end of file but not found.  eol-last (eslint)
```

However, it should be saying the error is on line 6.
